Upstream-Status: Backport
Signed-off-by: Ross Burton <ross.burton@intel.com>

From 56225ecca4f9598c0c9dbd7c46a51dd4816a383f Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Fri, 20 Apr 2018 13:39:15 -0700
Subject: sdiff: port to mingw
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Problem reported by Ross Burton (Bug#31218).
* src/sdiff.c (checksigs): Use ‘raise’, not ‘kill’.
---
 src/sdiff.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/sdiff.c b/src/sdiff.c
index 1b23a0c..b08bc74 100644
--- a/src/sdiff.c
+++ b/src/sdiff.c
@@ -805,7 +805,7 @@ checksigs (void)
 
       /* Yield an exit status indicating that a signal was received.  */
       untrapsig (s);
-      kill (getpid (), s);
+      raise (s);
 
       /* That didn't work, so exit with error status.  */
       exit (EXIT_TROUBLE);
-- 
cgit v1.0-41-gc330
